benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1696,6 +1696,12 @@ void GraphicsBenchmarkApp::RecordCommandBufferFullscreenQuad(PerFrame& frame, si
             frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[1], 3, &colorValues);
             break;
         }
+        case FullscreenQuadsType::FULLSCREEN_QUADS_TYPE_TEXTURE: {
```

Comment:
fyi, the reason for adding bracket (i.e. `case ... : { ... }`) to case statement is to limit variable scope, it's not necessary to add it to a break statement. It's also okay to leave it there if you prefer it

Reply:
Thanks, I didn't know this! I took it out